figures/mapping-histology-labels.Rmd

```diff
@@ -103,7 +103,55 @@ included_labels_df <- histologies_df %>%
 included_labels_df
 ```
 
-So the unique values for `broad_histology` and `cancer_group` above are what we need to take into account for our palette.
+### Collapsing "Other" categories
+
+In a later step, we will recode certain cancer groups, e.g., "High-grade glioma astrocytoma" will become "Other high-grade gliomas."
+There are other cancer groups that would fit in that category despite not meeting the sample size threshold above.
+For now, this applies to LGG, HGG, and embryonal tumors.
+
+```{r}
+other_cg_df <- histologies_df %>%
+  filter(sample_type == "Tumor",  # Exclude normal samples
+         !is.na(cancer_group),
+         broad_histology %in% c("Diffuse astrocytic and oligodendroglial tumor",
+                                "Low-grade astrocytic tumor",
+                                "Embryonal tumor")) %>%
+  select(broad_histology, 
+         cancer_group) %>% 
+  distinct() %>%
+  # Drop everything that meets the sample size requirement above
+  anti_join(included_labels_df)  
+
+other_cg_df
+```
+
+We'll create some vectors we'll use in later steps to group different cancer groups under the same "Other" cancer group display category.
+
+```{r}
+# To become "Other low-grade gliomas"
+other_lgg <- c(
+  "Low-grade glioma astrocytoma",
+  other_cg_df %>%
+    filter(broad_histology == "Low-grade astrocytic tumor") %>% 
+    pull(cancer_group)
+)
+
+# To become "Other high-grade gliomas"
+other_hgg <- c(
+  "High-grade glioma astrocytoma",
+    other_cg_df %>%
+    filter(broad_histology == "Diffuse astrocytic and oligodendroglial tumor") %>% 
+    pull(cancer_group)
+)
+
+# To become "Other embryonal tumors"
+other_embryonal <- c(
+  "CNS Embryonal tumor",  # Has n >= 10, but we're trying to limit # of colors
+  other_cg_df %>%
+  filter(broad_histology == "Embryonal tumor") %>%
+  pull(cancer_group)
+)
+```
```

Comment:
Can we print these three vectors at the end of this chunk? Helpful for following along in a notebook.

Reply:
I did something else that I prefer, but I believe still addresses your comment.

Reply:
I'm not sure I see what/where you did this something else?

Reply:
I agree with where you printed!